src/conf.py

```diff
@@ -125,6 +125,7 @@ def json_load(path):
 
 # NOTE: do not move to /tmp
```

Comment:
yes, `/tmp` is emptied on ec2 `stop` and `start`, undermining the point of a cache and spamming Glencoe unnecessarily.  

perhaps a `/var/cached/` in builder-base ... ?